docs/docs-content/clusters/public-cloud/azure/azure-disk-encryption.md

```diff
@@ -0,0 +1,353 @@
+       Encryption Set is displayed at the top.
+
+    5. Click the **Copy to clipboard** icon for the Resource ID and paste it into the `diskEncryptionSetID` field in the
+       Palette YAML editor.
+
+    </details>
+
+    ```yaml
+    cloud:
+      azure:
+        diskEncryptionSetID: "/subscriptions/<subscriptionId>/resourceGroups/<resourceGroup>/providers/Microsoft.Compute/diskEncryptionSets/<diskEncryptionSet>"
+    ```
+
+12. Make any other changes that you need and click **Next layer**.
+
+13. Select the remaining profile layers to finish the configuration.
+
+You can now [create a new Azure IaaS cluster](./create-azure-cluster.md) with disk encryption enabled using this cluster
+profile.
+
+### Edit an Existing Cluster Profile
+
+#### Enable Disk Encryption on a Cluster Profile
+
+Use the following steps to enable disk encryption on an
+[existing cluster profile](../../../profiles/cluster-profiles/modify-cluster-profiles/modify-cluster-profiles.md).
+
+1. Log in to [Palette](https://console.spectrocloud.com).
+
+2. Ensure you are in the correct project scope.
+
+3. From the left **Main Menu**, select **Profiles** and click the cluster profile that you want to edit.
+
+4. Create a new version of your cluster profile:
+
+   1. Click the version drop-down menu next to the cluster profile name, and click **Create new version**.
+   2. Fill the **Version** field with a new version number.
+   3. Click **Confirm**.
+
+5. Select the Kubernetes layer to view the **Edit Pack** drawer.
+
+   :::warning
+
+   Ensure that you are using
+
+   <VersionedLink text="Palette eXtended Kubernetes (PXK)" url="/integrations/packs/?pack=kubernetes" /> as the Azure
+   Disk Encryption option is only available to this pack.
+
+   :::
+
+6. In **Values**, click the **\</\>** button to show the YAML editor.
+
+7. On the right-hand side, click the **Presets** drop-down menu, and select the **Enable Encryption Using
+   Customer-Managed Key** option.
+
+8. Scroll to the bottom of the YAML editor to view the additional configuration that was added.
+
+   ```yaml
+   cloud:
+     azure:
+       diskEncryptionSetID: ""
+   ```
+
+9. Fill in the `diskEncryptionSetID` with the Resource ID URI of your Disk Encryption Set.
+
+   <!-- prettier-ignore -->
+   <details>
+   <summary> How to find the Resource ID URI of your Disk Encryption Set </summary>
+
+   1. Log in to the [Azure Portal](https://portal.azure.com/).
+
+   2. In the search bar, look for **Disk Encryption Sets**. Click on the service when found.
+
+   3. Find your Disk Encryption Set from the list and click on it to view details.
+
+   4. On the **Overview** page, click **JSON View** in the **Essentials** section. The Resource ID for the Disk
+      Encryption Set is displayed at the top.
+   5. Click the **Copy to clipboard** icon for the Resource ID and paste it into the `diskEncryptionSetID` field in the
```

Comment:
we should also tell users to replace `<subscriptionId>` and `<resourceGroup>` with their information